tests/test-maximize.lua

```diff
@@ -219,6 +219,25 @@ local steps = {
 
         c:kill()
 
+        return true
+    end,
+    function()
+        if #client.get() > 0 then return end
+
+        -- Test if resizing requests work
+        test_client(nil,nil,nil,nil,nil,{resize={x=400, y=400}})
+
+        return true
+    end,
+    function()
+        if #client.get() ~= 1 then return end
+
+        local c = client.get()[1]
+
+        if c.width ~= 400 or c.height < 400 then return end
```

Comment:
`select(2, c:titlebar_top())` should give you the size of the titlebar
(or perhaps in a more readable form: `local _, size = c:titlebar_top()`)